src/includes/class_reservationPermissions.php

```diff
@@ -109,29 +122,32 @@ public function checkBuilding($id = null){
     }
   }
 
-  public function checkRoom($id = null){
+  // checkRoom
+  // ==================================================
+  // Name : Tracy McCormick
+  //
+  // Description:
+  // Function requires room ID.
+  // Check and sees if the room ID is listed in the reservePermission table
+  // returns true if found false otherwise
+  public function checkRoom($roomID = null){
     try {
         // test to see if Id is present and valid
-        if(!isnull($id) && !$this->validate->integer($id)){
+        if(!isnull($roomID) && !$this->validate->integer($roomID)){
             throw new Exception("Invalid ID provided.");
         }
 
         $sql = "SELECT * FROM `reservePermissions` WHERE roomID = ? AND resourceType = 3";
 
         // get the results of the query
-        $sqlResult = $this->db->query($sql, array($id));
+        $sqlResult = $this->db->query($sql, array($roomID));
 
         if ($sqlResult->error()){
             throw new Exception("ERROR SQL" . $sqlResult->errorMsg());
         }
 
         //check and see if permissions exist on the resource
-        if ($sqlResult->rowCount() < 1) {
-           return FALSE;
-        }
-        else {
-           return TRUE;
-        }
+        return ($sqlResult->rowCount() < 1);
```

Comment:
Thanks this has been corrected and I made the same changes to other functions to match.